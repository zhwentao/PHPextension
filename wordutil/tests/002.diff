001+ minit
001- string(28) "百度腾讯头条小米oppo百度腾讯头条小米oppo"
002+ string(28) "Rinit百度腾讯头条小米oppo百度腾讯头条小米oppo"