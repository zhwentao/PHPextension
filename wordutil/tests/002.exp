string(28) "百度腾讯头条小米oppo百度腾讯头条小米oppo"