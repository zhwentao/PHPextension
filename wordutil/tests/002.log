
---- EXPECTED OUTPUT
string(28) "百度腾讯头条小米oppo百度腾讯头条小米oppo"
---- ACTUAL OUTPUT
minit
string(28) "Rinit百度腾讯头条小米oppo百度腾讯头条小米oppo"
---- FAILED
