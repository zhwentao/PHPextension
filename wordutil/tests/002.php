<?php 
$i = 10000;
while($i--) {
    echo replace_word("[{$i}]oppo 百度，阿里巴巴，滴滴小米，头条baidu{$i}---"). "\r\n";
}
/*
	you can add regression tests for your extension here

  the output of your test code has to be equal to the
  text in the --EXPECT-- section below for the tests
  to pass, differences between the output and the
  expected text are interpreted as failure

	see php5/README.TESTING for further information on
  writing regression tests
*/
?>
