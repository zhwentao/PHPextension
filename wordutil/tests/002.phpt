--TEST--
Check for wordutil presence
--SKIPIF--
<?php # vim:ft=php
if (!extension_loaded("wordutil")) print "skip"; ?>
--FILE--
<?php 
var_dump(replace_word("百度阿里腾讯头条滴滴小米oppo"));
/*
	you can add regression tests for your extension here

  the output of your test code has to be equal to the
  text in the --EXPECT-- section below for the tests
  to pass, differences between the output and the
  expected text are interpreted as failure

	see php5/README.TESTING for further information on
  writing regression tests
*/
?>
--EXPECT--
string(37) "百度***腾讯头条******小米oppo"
