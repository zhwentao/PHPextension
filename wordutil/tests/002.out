minit
string(28) "Rinit百度腾讯头条小米oppo百度腾讯头条小米oppo"